build.gradle

```diff
@@ -11,6 +11,7 @@ apply plugin: 'java'
 dependencies {
     testCompile 'junit:junit-dep:4.10'
     testCompile 'org.mockito:mockito-core:1.8.5'
+    testCompile 'com.pushtorefresh.java-private-constructor-checker:checker:1.0.0'
```

Comment:
I question the need of this assertion, thus it doesn't seem a strong enough value to warrant a dependency.